tests/jballerina-unit-test/src/test/resources/test-src/execution/projectModuleDependencyChainForInit/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[project]
+org-name= "unit-tests"
```

Comment:
@warunalakshitha , can you check the naming used by @HindujaB here for org name and module names?

Reply:
We need to improve the module names based on coding best practices. Module names can contain alphanumeric characters including dots. Dots in a module name have no meaning other than the last segment after the final dot being used as a default alias within the source code. The underscore ( _ ) character and camel case are not allowed.

Ex: lang.string, auth - valid module names

Reply:
Sorry. Fixed it in recent commit.